app/[locale]/academics/departments/[name]/page.tsx

```diff
@@ -1,9 +1,392 @@
+        >
+          <section className="flex h-full flex-col justify-between space-y-7 lg:w-1/2">
+            <div className="flex flex-col">
+              <h2 className="m-0">
+                {text.sectionTitles[1].toLocaleUpperCase()}
+              </h2>
+              <p>
+                Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ipsum
+                quasi eaque enim cum esse doloremque magni explicabo quos,
+                assumenda beatae totam nesciunt sunt tenetur omnis dolorum nemo
+                cupiditate cumque corrupti provident doloribus! Blanditiis
+                quaerat itaque atque illo, modi est fugiat.
+              </p>
+            </div>
+            <div className="flex flex-col">
+              <h2 className="m-0">
+                {text.sectionTitles[2].toLocaleUpperCase()}
+              </h2>
+              <p>
+                Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ipsum
+                quasi eaque enim cum esse doloremque magni explicabo quos,
+                assumenda beatae totam nesciunt sunt tenetur omnis dolorum nemo
+                cupiditate cumque corrupti provident doloribus! Blanditiis
+                quaerat itaque atque illo, modi est fugiat.
+              </p>
+            </div>
+          </section>
+          <div className="relative hidden lg:block lg:h-full lg:w-1/2">
+            <Image
+              className="h-full w-full rounded-md"
+              alt={departmentData.name}
+              width={300}
+              height={400}
+              src="https://www.figma.com/file/2UXH7QR1dUT8KXgdGuYaC1/image/cf46d1812e6285efda1f8172b7fc3634c6d76130"
+            />
+          </div>
+        </section>
+
+        {/*  hod's message*/}
+        <section className="mt-20 rounded-md border border-primary-700 bg-neutral-50 p-5 lg:flex lg:space-x-4">
+          <section className="hidden w-full flex-col lg:flex lg:w-1/4">
+            <Image
+              src={
+                'https://www.figma.com/file/2UXH7QR1dUT8KXgdGuYaC1/image/cf46d1812e6285efda1f8172b7fc3634c6d76130'
+              }
+              alt="hod"
+              width={350}
+              height={450}
+              className="h-[295px] w-64 rounded-md xl:h-[300px] xl:w-96"
+            />
+            <div className=" lg:mt-4 lg:space-y-1">
+              <HodDetail
+                icon={MdEmail}
+                items={['jitenderchhabra@nitkkr.ac.in']}
+              />
+              <HodDetail
+                icon={FaPhoneAlt}
+                items={['+91-1744-233482 (Off-Direct)', '9416733789 (Mob)']}
+              />
+              <HodDetail
+                icon={FaRegCalendarAlt}
+                items={['Dated Academic Session 2010 - 2025']}
+              />
+            </div>
+          </section>
+          <div className="flex w-full flex-col space-y-20 lg:w-3/4">
+            <blockquote>
+              <div className="mb-5 flex items-center space-x-5">
```

Comment:
Don't add a gap here, you're misaligning the heading and the text. Add it on the parent `<section>` right under the `hod's message` comment.
```suggestion
              <div className="mb-5 flex items-center">
```